docker/runit/service/spark/run

```diff
@@ -31,6 +31,14 @@ function set_log_level() {
         ${SPARK_DIST_PATH}/conf/log4j.properties.template >${SPARK_DIST_PATH}/conf/log4j.properties
 }
 
+function set_statsd_address() {
+  if [ "$STATSD_UDP_HOST" != "" ] && [ "$STATSD_UDP_PORT" != "" ]; then
```

Comment:
Are these env vars injected by UCR?

Reply:
Yes, they're added by the metrics module in the mesos agent.